packages/node/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "strictNullChecks": true,
     "allowSyntheticDefaultImports": true,
     "outDir": "build/main",
-    "module": "commonjs",
+    "module": "CommonJS",
+    "moduleResolution": "NodeNext",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** What does moduleResolution add?‏

Reply:
`nodenext` modes integrate with Node’s [native ECMAScript Module support](https://nodejs.org/api/esm.html) and if not mistaken we also get subpath exports.